[2024-06-13 20:29:45,936] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_ml_to_postgres.transform_data manual__2024-06-13T20:24:36.172682+00:00 [queued]>
[2024-06-13 20:29:45,939] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_ml_to_postgres.transform_data manual__2024-06-13T20:24:36.172682+00:00 [queued]>
[2024-06-13 20:29:45,939] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-06-13 20:29:45,940] {taskinstance.py:1357} INFO - Starting attempt 2 of 2
[2024-06-13 20:29:45,940] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-06-13 20:29:45,945] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): transform_data> on 2024-06-13 20:24:36.172682+00:00
[2024-06-13 20:29:45,948] {standard_task_runner.py:52} INFO - Started process 5041 to run task
[2024-06-13 20:29:45,949] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 's3_to_ml_to_postgres', 'transform_data', 'manual__2024-06-13T20:24:36.172682+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/ml_dataset_preparation.py', '--cfg-path', '/tmp/tmpe6cj52m3', '--error-file', '/tmp/tmp6qtfmeup']
[2024-06-13 20:29:45,950] {standard_task_runner.py:80} INFO - Job 13: Subtask transform_data
[2024-06-13 20:29:45,970] {task_command.py:370} INFO - Running <TaskInstance: s3_to_ml_to_postgres.transform_data manual__2024-06-13T20:24:36.172682+00:00 [running]> on host 92769061d4a4
[2024-06-13 20:29:45,996] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=s3_to_ml_to_postgres
AIRFLOW_CTX_TASK_ID=transform_data
AIRFLOW_CTX_EXECUTION_DATE=2024-06-13T20:24:36.172682+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-13T20:24:36.172682+00:00
[2024-06-13 20:29:46,043] {logging_mixin.py:115} INFO - Transformed data saved to /opt/***/data/transformed_data/transformed_data.csv
[2024-06-13 20:29:46,044] {python.py:173} INFO - Done. Returned value was: None
[2024-06-13 20:29:46,050] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=s3_to_ml_to_postgres, task_id=transform_data, execution_date=20240613T202436, start_date=20240613T202945, end_date=20240613T202946
[2024-06-13 20:29:46,097] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-06-13 20:29:46,113] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
